lib/elixir/lib/file.ex

```diff
@@ -89,6 +89,9 @@ defmodule File do
   @doc """
   Returns `true` if the path is a regular file.
 
+  This function follows symbolic links, so if the symbolic link points to a
+  regular file, this also returns `true`.
```

Comment:
", it returns `true`." or ", `true` is returned." reads better to me, what do you think?